### PR TITLE
Remove Google Analytics direct integration

### PR DESCRIPTION
**Pull request by atduarte**
Google Analytics will happen via Segment.

cc @jakobhero 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/760"></a>

### CODE REVIEW

**Comment by atduarte** [comment]
As we are integrating Google Analytics via Segment, Google Analytics shouldn't be able to use their own cookies. As such removed that part of the Privacy Policy. @jakobhero please review that part as I'm unsure.

File: src/routes/$layout.svelte [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <script>
   import Segment from "../components/segment.svelte";
-  import GoogleAnalytics from "../components/google-analytics.svelte";
```
Comment (mikenikles) [inline src/routes/$layout.svelte:3]: ❤️

**Review by mikenikles — APPROVED** [review]
The removal of Google Analytics looks good to me. As per Slack conversation, please coordinate the release with Jakob to make sure data flows from Segment to Google Analytics.

**Review by jakobhero — APPROVED** [review]
The update to the privacy policy looks good to me.

**Pull request merged**